### ISSUE TITLE
some releases untagged

### ISSUE THREAD

**Issue by nschloe** [trilinos/Trilinos#42]
On https://github.com/trilinos/Trilinos/releases, we have a good overview of the tags. The current release 12.4.2 and also 12.4.0 are missing. There may be more.

**Comment by bmpersc** [comment]
There was no 12.4.0 tag because there was no 12.4.0 tarball. There is the initial-trilinos-release-12-4-branch tag though, which would be the closest thing to 12.4.0.
The 12.4.2 it looks like that was just missed from the script that was used to push up all the tags. I'll manually add it.

**Comment by bmpersc** [comment]
There was one other tag missing 12.2.2. It is no surprise that 12.4.2 and 12.2.2 were releases we pushed out just before the move to github in order to not need to deal with the issues the move/filtering will have on our release process. That likely meant the list of what tags needed to go out was generated before those releases were done. Thankfully the candidate still had them so it was an easy fix. I have pushed both up to Trilinos now.
